e2e/tests/attendance-tests.spec.ts

```diff
@@ -269,6 +269,11 @@ test("Assign a recurring activity to a user", async ({ page }) => {
   // When I assign myself
   await page.getByRole("button", { name: "Edit" }).click();
   await page.getByLabel("Assigned user(s)").click();
+
+  // wait to ensure the dropdown option is rendered and interactable, to avoid playwriting timeout issue
+  await page
+    .getByRole("option", { name: currentUser.name, exact: true })
+    .waitFor({ timeout: 10000 });
```

Comment:
@geigerzaehler , any ideas why this apparently started running into timeouts?
And is this the best way to solve it or can/should we add timeout to the click?